src/load/azext_load/data_plane/load_test/params.py

```diff
@@ -29,6 +29,9 @@ def load_arguments(self, _):
         c.argument("engine_instances", argtypes.engine_instances)
         c.argument("custom_no_wait", argtypes.custom_no_wait)
         c.argument("disable_public_ip", argtypes.disable_public_ip)
+        c.argument("autostop", argtypes.autostop)
```

Comment:
what are the possible values of auto-stop ? enable/disable

Reply:
yes, the help text for arguments are present in argtypes.py